### PR TITLE
Disabled checked checkbox css rules

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix https://github.com/owncloud/core/pull/20530#issuecomment-162439357

@MorrisJobke @jancborchardt @Henni 

Please review

### CODE REVIEW

File: core/css/inputs.css [hunk 1]
```diff
@@ -108,10 +108,28 @@ html:not(.ie8) input[type="checkbox"].checkbox:checked + label:before {
...
+
 html:not(.ie8) input[type="checkbox"].checkbox--white:checked + label:before {
 	background-image: url('../img/actions/checkbox-checked-white.svg');
 }
 
+html:not(.ie8) input[type="checkbox"].checkbox--white:disabled + label:before {
+	background-image: url('../img/actions/checkbox-disabled-white.svg');
+}
+
+/* TODO icon is missing
```
Comment (nickvergessen) [inline core/css/inputs.css]: This image is missing, should it use `checkbox-checked-disabled.svg` since it has no colour but is grey, it might fit aswell?

Comment (jancborchardt) [inline core/css/inputs.css]: Yes, that was the intention. :)

**Comment by MorrisJobke** [comment]
Tested and works :+1:

File: core/css/inputs.css (hunk 1 above)
Comment (nickvergessen) [inline core/css/inputs.css]: fixed, please review

**Comment by Henni** [comment]
Looks good :+1:

image

**Pull request merged**